Cloudinary/Api.cs

```diff
@@ -77,7 +77,7 @@ static Api()
         /// <param name="parameters">Dictionary of call parameters (can be null)</param>
         /// <param name="file">File to upload (must be null for non-uploading actions)</param>
         /// <returns>HTTP response on call</returns>
-        public HttpWebResponse Call(HttpMethod method, string url, SortedDictionary<string, object> parameters, FileDescription file, Dictionary<string, string> extraHeaders = null)
+        public override object Call(HttpMethod method, string url, SortedDictionary<string, object> parameters, FileDescription file, Dictionary<string, string> extraHeaders = null)
```

Comment:
This method was initially publicly available, but it shouldn't be this way. API users shouldn't call it directly, it's our internal method. Everything that users should be able to call can be found in Cloudinary's class, otherwise it breaks our architecture. Let's to make this method private, so that no one but us would be able to use it.